app/functions.R

```diff
@@ -120,44 +119,46 @@ renderScoreHtml <- function(risk) {
   )))
 }
 
-renderVaccinesHtml <- function(go, has_vaccine, vaccine, doses, days){
-  validate(need(go, ""))
+renderVaccinesHtml <- function(risk, has_vaccine, vaccine, months_last_vaccination){
   
   # Case where not vaccinated
   if (!has_vaccine){
     # no doses
-    text <- tags$p("There are currently", total_vaccines_str, "vaccines against COVID-19 authorized for use in the United States. ",
-           "All of the approved vaccines are safe and highly effective at preventing symptomatic COVID-19. ", 
+    text <- tags$p("There are currently vaccines against COVID-19 authorized for use in the United States. 
```

Comment:
I am okay with removing the "total_vaccine_str" from the output, but grammatically should we add something here? We can either hardcode "3" because we are talking about the US, or we can change the sentence to "8 vaccines approved by the WHO" and cite this page: https://covid19.trackvaccines.org/agency/who/

Reply:
Hmmm, good point, I see what you are saying. I think that this sentences is technically correct thought I can see how it might be awkward. @mollybair  What do you think about the sentence flow?

Reply:
I agree that it would be good to include the WHO citation.